src/calculateRentalCost.js

```diff
@@ -4,7 +4,12 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
+    return days * 40 - 20;
+  } else {
+    return days * 40 - 50;
```

Comment:
Consider using descriptive constant names for the number of days threshold and discount amounts instead of hardcoding the values. This will make the code easier to understand and maintain.